### PR TITLE
Download the toitdoc-web-page and pkg-web-page from GitHub.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

**Comment by floitsch** [comment]
TBR.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,10 @@
 
+# The toitdoc page used to server package toitdocs.
```
Comment (kasperl) [inline Makefile:2]: I don't really understand this sentence.

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,10 @@
```
Comment (kasperl) [inline Makefile:1]: Drop empty first line?

**Review by kasperl — COMMENTED** [review]
Looks good to me.

**Pull request merged**